### PR TITLE
fix upper constraints by vendoring file

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: roles/openstack-source/files/constraints.txt [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+pyghmi===1.0.9
+pyinotify===0.9.6
+pykerberos===1.1.13;python_version=='2.7'
+pyldap===2.4.25.1
+pylxd===2.0.3
+pymemcache===1.3.6
+pymod2pkg===0.5.4
+pymongo===3.3.0
+pyngus===2.1.2
+pyparsing===2.1.8
```
Comment (zfeldstein) [inline roles/openstack-source/files/constraints.txt]: Should this be pyparsing == 2.10.0

File: .gitignore [hunk 2]
```diff
@@ -13,3 +13,5 @@ ursula.log
 openstack
 *.swp
 *.swo
+.ssh_config
```
Comment (zfeldstein) [inline .gitignore]: 👍

File: roles/openstack-source/files/constraints.txt (hunk 1 above)
Comment (paulczar) [inline roles/openstack-source/files/constraints.txt]: luuuulz yes :(

**Comment by paulczar** [comment]
retest

**Comment by nirajdp76** [comment]
LGTM. Once upstream bug is resolve do we plan to get rid of vendored  constraints.txt as it will get outdated soon?

**Pull request merged**